lms/djangoapps/verify_student/models.py

```diff
@@ -944,6 +966,18 @@ def verification_status_for_user(cls, user, course_id, user_enrollment_mode):
         else:
             return 'ID Verified'
 
+    @classmethod
+    def is_verification_expiring_soon(cls, expiration_datetime):
+        """
+        Returns True if verification is expiring within VERIFICATION_EXPIRATION_DAYS.
+        """
+        if expiration_datetime:
+            if (expiration_datetime - datetime.now(pytz.UTC)).days <= settings.VERIFY_STUDENT.get(
+                    "VERIFICATION_EXPIRATION_DAYS", 28):
```

Comment:
Specifying a default value for a setting in code like this is an anti-pattern. Please don't do this. It leads to surprising behavior. If you need 28 to be the default, make that the default value of the setting.